docs/calculations.mdx

```diff
@@ -173,6 +173,10 @@ From [Carbon impact of video streaming (Carbon Trust)](https://ctprodstorageacco
 
 <PowerModelDefaults />
 
+To determine network when it's not submitted we default to fixed for pc, tv and smart speakers.
+When we don't know the device or the device is a phone or tablet we calculate network costs as a weighted average of the
+fixed and mobile usage in the country (if available) or worldwide.
+
```

Comment:
can we also update the fallback snippet to make clear how this works? https://methodology.scope3.com/calculations#fallbacks

Reply:
now that you mention it that looks like a way better place for this explanation (and the defaults) moving and expanding it now